base/array.jl

```diff
@@ -1494,7 +1499,7 @@ function sizehint!(a::Vector, sz::Integer)
 end
 
 # Fall-back implementation for non-shrinkable collections
-_sizehint!(a, sz; shrink) = sizehint!(a, sz)
+sizehint!(a, sz; shrink::Bool) = sizehint!(a, sz)
```

Comment:
Shouldn't this be `sizehint!(a, sz; shrink::Bool=true)`? Otherwise AFAICT this will throw if `shrink` isn't passed?

Reply:
This is a fall-back for user-defined collections. If someone tries to use `shrink` it will not throw, but call the old implementation of `sizehint!`.